src/Block_Reservoir.f90

```diff
@@ -1,28 +1,32 @@
-module Block_Reservoir
+Module Block_Reservoir
+
+
 
         implicit none
-        
+
         ! --------------------------------------------------------------------
         !       reservoir to RBM variables
         !----------------------------------------------------------------------
 
-        !------------------------read in reservoir info--------------------------
-        real, dimension (:), allocatable:: dam_lat, dam_lon, res_grid_lat, res_grid_lon
-        real, dimension (:), allocatable:: res_depth_feet, res_width_feet, res_length_feet
+        !------------------------read in reservoir
+        !info--------------------------
+        real, dimension (:), allocatable:: dam_lat, dam_lon, res_grid_lat,res_grid_lon
+        real, dimension (:), allocatable:: res_depth_feet, res_width_feet,res_length_feet
         integer, dimension (:), allocatable:: dam_number, start_operating_year
-        real, dimension (:), allocatable:: res_top_vol, res_bot_vol, res_max_flow, res_min_flow
+        real, dimension (:), allocatable:: res_top_vol, res_bot_vol,res_max_flow, res_min_flow
         integer, dimension (:), allocatable:: res_start_node, res_end_node
-       !  integer, dimension (:,:), allocatable:: nodes_x   !for each reach, what are all the nodes
+       !  integer, dimension (:,:), allocatable:: nodes_x   !for each reach,
+       !  what are all the nodes
```

Comment:
In line 13, 14 and 16, add an blank between comma and next variable name.

Delete comment line 18 and line 19